app/experimenter/experiments/email.py

```diff
@@ -60,3 +60,25 @@ def send_intent_to_ship_email(experiment_id):
         cc=[experiment.owner.email],
     )
     email.send(fail_silently=False)
+
+
+def send_experiment_launch_email(experiment):
+
+    content = render_to_string(
+        "experiments/launch_experiment_email.txt", {"experiment": experiment}
```

Comment:
I'm so glad we're doing this text file email template thing, it's such a nice little pattern.  Maybe it would be helpful to put them in a subfolder like `experiments/emails/`?

Reply:
good idea.